src/js/types/lastIngestionDataTypeResponse.ts

```diff
@@ -0,0 +1,9 @@
+export interface LastIngestionDataTypeResponse {
+  count: number | null;
+  id: string;
+  label: string;
+  last_ingested: string | null;
+  queryable: boolean;
+}
+
+export type DataResponseArray = LastIngestionDataTypeResponse[];
```

Comment:
is this file used anywhere? `LastIngestionDataTypeResponse` is duplicate here - you already defined `DataTypeResponse` in the store